### ISSUE TITLE
Open new tab/windows with current working directory

### ISSUE THREAD

**Issue by Nudin** [kovidgoyal/kitty#952]
Several terminals (Konsole, terminator, etc) have the behavior that when you open a new tab (or window) the new tab will have the same working directory as the currently active terminal. I got used to this a lot and like it, so I miss it in kitty. Could an option be added to activate this behavior?

**Comment by erf** [comment]
```map cmd+t new_tab_with_cwd```

**Comment by Nudin** [comment]
By reading the code I found out that it is already possible:
```
map kitty_mod+t new_tab_with_cwd
map kitty_mod+enter new_window_with_cwd
```

**Comment by p-dr** [comment]
`map kitty_mod+enter new_os_window_with_cwd` also works for OS windows.

**Comment by LucasBR96** [comment]
> map kitty_mod+t new_tab_with_cwd
map kitty_mod+enter new_window_with_cwd

This solution by @Nudin didn't work for me.

**Comment by ujjwal3067** [comment]
Worked for me. You do have to restart the terminal to take effect.

**Comment by vkocubinsky** [comment]
I set 

```
map kitty_mod+enter new_window_with_cwd
map kitty_mod+n new_os_window_with_cwd
map kitty_mod+t     new_tab_with_cwd
```

But New Tab, New Window, New OS Window doesn't work for macOS Big sur.

**Comment by ant-fx** [comment]
@vkocubinsky seems to be working for me on MacOS Monterey.

Try making sure you fully restart Kitty to reload the config. I didn't think it was working at first because I'd exited all the terminal sessions but that didn't actually close the app.

**Comment by vkocubinsky** [comment]
Thanks for comment, yes I am sure that I fully restart. Because you get me know that feature works I found the way, and find the root cause. I found that mapping with "cmd+t" works
```
map cmd+t     new_tab_with_cwd
```
But mapping with "kitty_mod" doesn't works
```
map kitty_mod+t     new_tab_with_cwd
```
Then I put 
```
kitty_mod cmd
```
And now everything works include mapping with "kitty_mod"! Probably kitty_mod is not set to cmd for macos by default. Thanks for help.
